src/lib/Microsoft.Health.Fhir.Ingest.Template/FhirCodeableConcept.cs

```diff
@@ -12,6 +13,7 @@ public class FhirCodeableConcept
         public string Text { get; set; }
 
 #pragma warning disable CA2227
+        [JsonProperty(Required = Required.Always)]
         public IList<FhirCode> Codes { get; set; }
```

Comment:
I don't think this is correct.  The CodeableConcept in FHIR allows 0 to N codings and 0 to 1 text element.

https://www.hl7.org/fhir/datatypes.html#CodeableConcept so it is valid for someone to supply a text element but not supply any codes.

Reply:
Same comment applies to CodeableConceptFhirValueType.cs above.

Reply:
Thanks for your feedback. I've further explored the CodeableConcept docs and see that 0-N codings are allowed. I've removed the required attribute on Codes from FhirCodeableConcept.cs and CodeableConceptFhirValueType.cs